inventory/group_vars/ch.yml

```
@@ -0,0 +1,23 @@
+---
+
+# See https://github.com/openfoodfoundation/ofn-install/wiki/Setup for more info
+checkout_zone: Europe
+country_code: CH
+currency: CHF
+locale: en                  # Default locale for OFN instance
+available_locales: de_CH,fr_CH,it_CH,en_CH    # Available locales for OFN instance
+language: en_US.UTF-8       # Default language for the server
+language_packages:          # Language packages for the server
+  - language-pack-en-base
+l10n_repo: https://github.com/openfoodfoundation/l10n_ch.git
+timezone: Vienna
+
+
+# Used for the LetsEncrypt email setting
+developer_email: mikel.cordovilla@tegonal.com
+
+# Users listed here will have their key from /files/keys added to the app_user
+users_sysadmin:
+  - mikel
+  - matt
```

Why only Matt? He's actually not working on OFN at the moment. If you want us to be able to deploy:
```suggestion
  - mikel
  - "{{ core_devs }}"
```